tool/tctl/common/token_command.go

```diff
@@ -220,6 +220,26 @@ func (c *TokensCommand) Add(ctx context.Context, client auth.ClientI) error {
 
 	// Print signup message.
 	switch {
+	case roles.Include(types.RoleWindowsDesktop):
```

Comment:
If the token includes both desktop and kube roles, you'll only print the destkop template here. Is this the behavior you intended?

Reply:
They're all like this depending on the order. You only get one of them.

Reply:
We can change it to iterate through and print all the situations

Reply:
We can move desktop to the end.

Reply:
I prefer the desktop output that just links to docs for using the token instead of printing commands to use (but I have other issues with `tctl token add`, see: https://github.com/gravitational/teleport/issues/19894).

I think your PR is fine as-is because it's just doing the same as other types of tokens, but how would you feel about changing `tctl tokens add` to print a list of docs links for each token type in the `--token` list? For example:

```
$ tctl tokens add --type=kube,windows_desktop
The invite token: 124ffa40c84cc7ea0d855b9fe892176e
This token will expire in 60 minutes.

This token enables kubernetes and windows desktop services.
See Teleport documentation for more details:
- https://goteleport.com/docs/kubernetes-access
- https://goteleport.com/docs/desktop-access/

Reply:
@GavinFrazar I'm fine with changing to that.  I think the output has caused more confusion in cases.

Reply:
@zmb3 thoughts?

Reply:
I'm fine with just moving desktop to the end.

Reply:
@zmb3 moved

Reply:
@stevenGravy thanks, I just wanted your thoughts on that suggestion. If it wasn't clear, I should clarify that I wasn't suggesting you make that change (to use docs links only) in this PR.

Reply:
@zmb3 good to go then?